modules/nlu/src/backend/dialog-conditions/intent-ambiguous.ts

```diff
@@ -0,0 +1,41 @@
+import { Condition, IO } from 'botpress/sdk'
+import _ from 'lodash'
+
+const AMBIGUITY_GAP = 0.15
+
+export default {
+  id: 'intent-is-ambiguous',
+  label: 'Intent is ambiguous within topic',
+  description: `The users's intention is can be interpreted as multiple intents within the same topic`,
+  displayOrder: 1,
+  evaluate: (event: IO.IncomingEvent) => {
```

Comment:
Do you think AMBIGUITY_GAP is something that may be useful to be configurable? If so, you should set it as a parameter with a default value

Reply:
parameter, yes that is it !